myScript.js

```diff
@@ -0,0 +1,75 @@
+const bookList = document.getElementById('bookListItems');
+const form = document.getElementById('form');
+const removebtn = document.getElementsByClassName('removebtn');
+const newFormBook = document.getElementById('newFormBook');
+const newFormAuthor = document.getElementById('newFormAuthor');
+let books = [];
+const findBooks = () => {
+  if (localStorage.getItem('bookList') !== []) {
+    books = JSON.parse(localStorage.getItem('bookList'));
+  } else {
+    books = [];
+  }
+  return books;
+};
```

Comment:
Just as the previous reviewer mentioned about looking over an empty array. The problem here is that, when trying to display books, the find method finds the bookLists from localStorage, which is not there at the initial start of the app, and for that, it gives cannot read properties of null because there is no bookList initially when that app starts. It's fixed easily with the following key points:

<img width="842" alt="Screenshot 2023-01-11 at 8 51 55 AM" src="https://user-images.githubusercontent.com/50941074/211762207-14581e06-ff15-4d55-9a71-8be54b299dc5.png">

- [ ] It would be nice if you could delete these highlighted codes and then declare the books variable, set its value to an empty array, and at the same time get the localStorage books in case it's available. And this is done in just one line below:

- `let books = JSON.parse(localStorage.getItem("bookList")) || [];`

- [ ] Then after this fix, you could remove the findBooks method used in the `widow.onload` at line 66 method, because you're already getting the books from the variable at the top. Please try it, it will fix things up 👍🏽

Reply:
This was a great review and thanks to your precise review I was able to solve everything I am extremely thankful